modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/transaction/TransactionValidator.scala

```diff
@@ -29,12 +33,17 @@ object TransactionValidator {
     stardustPrimary
   )
 
+  private val addressesLockedAtOrdinal: Map[Address, SnapshotOrdinal] = Map(
+    Address("DAG6LvxLSdWoC9uJZPgXtcmkcWBaGYypF6smaPyH") -> SnapshotOrdinal(1447110L) // NOTE: BitForex
+  )
+
   def make[F[_]: Async](
     signedValidator: SignedValidator[F]
   ): TransactionValidator[F] =
     new TransactionValidator[F] {
       def validate(
-        signedTransaction: Signed[Transaction]
+        signedTransaction: Signed[Transaction],
+        maybeOrdinal: Option[SnapshotOrdinal] = None
```

Comment:
If such validation needs some context (like ordinal from snapshot storage) then it should be in `ContextualTransactionValidator` probably 🤔

Reply:
@kpudlik There is already code in there to exclude the locked address. I thought these changes are for a different situation, that is, meant to be executed from the block validator.

Reply:
Contextual only blocks the L1, previous patch already implemented it there, so it'll reject transactions at L1 layer on submission, this change should only really affect block validation in that sense.